src/decorator.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import * as _helpers from './helpers';
+import * as protos from '../protos/protos';
+
+const enhancedTypesJson = require('./enhanced-types.json');
+
+// Walk the tree of nested namespaces contained within the enhanced-types.json file
+function walkNamespaces(
+  // tslint:disable-next-line no-any
+  jsonNode: Record<string, any>,
```

Comment:
would it silence these warnings if we we gave a slightly more specific type hint like `Record<string, object>`?

Reply:
Changed.

Unfortunately no, setting those to `Record<string, object>` didn't work.

I tried to provide define / provide types as much as possible in the recent commits. However, in some cases I needed to cast to `unknown` before casting to another type. In two specific cases, I wasn't able to work around using any--both uses are deep within the bowels of internal functions.  (Even so, I'll continue to noodle on ways to address this issue.)